Dashboards, typical values and notebooks deletion

ArturMakowski opened the pull request:
- First dashboards class prototype with a few dashboards
- Typical values calculator method
- Notebook deletion from git tracking

pdGruby commented on app/dashboards.py:
```diff
@@ -0,0 +1,146 @@
+import pandas as pd
+import plotly.express as px
```
Needs to be added to the requirements.txt

pdGruby commented on ml_model/notebooks/.gitignore:
We already have .gitignore file in the root directory, so it should be added there

pdGruby commented on ml_model/pricepy_model.py:
```diff
@@ -112,12 +113,49 @@ def fit(self):
+    def data_without_na(self):
+        return self.data.dropna()
+
+    def get_typical_values(self):
```
This is a very non-optimized approach. We call this method when the model training is done, and there is no need of calling this method when we do nothing after the data that is computed.

Once the model training is done & the model is saved to the database, we completely lose the data that we have just computed. After that, when we want to use our model for predictions and allow users not to fill all model's features, we have to call this method in the application. Then, the data that we compute is finally useful and can be used when the user do not provide all features to the model.

In this approach, every user that enters our application will trigger the "get_typical_values" method and the same operation will be performed for every user. It may lead to the problems with resources. What is more, this is very unhandy for the developers.

We should call this method and compute the data when we train the model - that's right. But after that, we need a place to store the data - for example, we can store it in the database table. When we store such data, then we can improve our "predict" method & tell it to fill "None" values with the most typical values. This way we have optimized approach & a handy 'predict' method for the developers.

pdGruby reviewed (changes requested):
Please, see the comments in the code

ArturMakowski commented:
> Please, see the comments in the code

- Implemented none values mapping in predict method
- Fixed some issues with time series dashboards, it now shows one location trace by default and you can add traces for more locations or change it to other location with dropdown menu
- Addressed typical values issues by assigning them to sklearn model attribute as dict and retrieving it when loading model pickle from database (IMHO there is no need to create separate table in database for typical values, we can just store them in model object pickle as attribute (dict) that we store in database, because typical values dict is small and it will always be attached to our model pickle, there will be no worry of syncing them later)
